variables.tf

```diff
@@ -142,6 +142,12 @@ variable "efs_mount_points" {
   description = "The mount points for data volumes in your container. This parameter maps to Volumes in the --volume option to docker run"
 }
 
+variable "efs_posix_user" {
+  type        = list(number)
```

Comment:
Can you change this to another type? Now somebody can pass `[1]` and everything breaks. If it's a fixed setup (UID and GUID), just change the variable type in something explicit

Reply:
Like this?

variable "efs_posix_user" {
  type        = number
  default     = 1000
  description = "Posix uid needs to be mapped at EFS Access Point"
}

variable "efs_posix_group" {
  type        = number
  default     = 1000
  description = "Posix gid needs to be mapped at EFS Access Point"
}